contents/blog/best-mobile-app-ab-testing-tools.mdx

```diff
@@ -0,0 +1,303 @@
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+
+[Optimizely](/blog/posthog-vs-optimizely) is an all-in-one system for marketing that includes A/B testing **and** more – like content management, campaign planning, asset management, and checkout customizations. 
+
+It's built for marketers in large enterprises looking to optimize their content, apps, and e-commerce experiences.
+
+### Which platforms does Optimizely support?
+
+<ComparisonTable column1="Optimizely">
+  <ComparisonRow column1={true} feature="Android" description="" />
+  <ComparisonRow column1={true} feature="iOS" description="" />
+  <ComparisonRow column1={true} feature="React Native" description="" />
+  <ComparisonRow column1={true} feature="Flutter" description="" />
+  <ComparisonRow column1={true} feature="Web" description="" />
+  <ComparisonRow column1={true} feature="Server" description="" />
+</ComparisonTable>
+
+### How much does Optimizely cost?
+
+Optimizely's pricing is sales-driven and completely opaque, so it's not clear how much it costs. However, according to [Splitbase](https://splitbase.com/blog/optimizely-pricing), Optimizely will cost you at least $36,000 per year.
+
+## 4. Firebase
+
+![Firebase mobile app A/B tests](../images/blog/best-mobile-app-ab-testing-tools/firebase.png)
+
+- **Best for:** Apps already using Firebase's other features
+- **Free tier:** <span className="text-green text-lg">✔</span>
+- **Transparent pricing:** <span className="text-green text-lg">✔</span>
+- **Open source:** <span className="text-red text-lg">✖</span>
+- **Get started without talking to sales:** <span className="text-green text-lg">✔</span>
+- **Includes product analytics:** <span className="text-red text-lg">✖</span>
+
+[Firebase A/B testing](https://firebase.google.com/products/ab-testing) is built specifically for mobile apps and is completely free. It includes an easy to use interface for running experiments on Firebase's other app features, such as [Cloud Messaging](https://firebase.google.com/products/cloud-messaging) and [In-App Messaging](https://firebase.google.com/products/in-app-messaging). 
+
+It's important to note that using A/B testing on Firebase requires installing [Google Analytics](/blog/posthog-vs-ga4) in your app. It's also not possible to run A/B tests on your web app or server.
+
+### Which platforms does Firebase support?
+
+<ComparisonTable column1="Firebase">
+  <ComparisonRow column1={true} feature="Android" description="" />
+  <ComparisonRow column1={true} feature="iOS" description="" />
+  <ComparisonRow column1={false} feature="React Native" description="" />
+  <ComparisonRow column1={false} feature="Flutter" description="" />
+  <ComparisonRow column1={false} feature="Web" description="" />
+  <ComparisonRow column1={false} feature="Server" description="" />
+</ComparisonTable>
+
+### How much does Firebase A/B testing cost?
+
+A/B testing with Firebase is completely free. 
```

Comment:
What does Firebase cost more generally?

Reply:
It depends on what you use them for, but you can use their A/B testing suite for free without using any of their other products